components/org.apache.stratos.cli/src/main/java/org/apache/stratos/cli/commands/AddNetworkPartitionCommand.java

```diff
@@ -0,0 +1,174 @@
+
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.apache.stratos.cli.commands;
+
+import org.apache.commons.cli.*;
+import org.apache.stratos.cli.Command;
+import org.apache.stratos.cli.RestCommandLineService;
+import org.apache.stratos.cli.StratosCommandContext;
+import org.apache.stratos.cli.exception.CommandException;
+import org.apache.stratos.cli.utils.CliConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class AddNetworkPartitionCommand implements Command<StratosCommandContext> {
+
+    private static final Logger logger = LoggerFactory.getLogger(AddNetworkPartitionCommand.class);
+
+    private final Options options;
+
+    public AddNetworkPartitionCommand(){
+        options = constructOptions();
+    }
+
+    private Options constructOptions() {
+        final Options options = new Options();
+
+        Option resourcePath = new Option(CliConstants.RESOURCE_PATH, CliConstants.RESOURCE_PATH_LONG_OPTION, true,
+                "Network partition resource path");
+        resourcePath.setArgName("resource path");
+        options.addOption(resourcePath);
+
+        return options;
+    }
+    /**
+     * @return The name of the command
+     */
+    @Override
+    public String getName() {
+        return "add-network-partition";
+    }
+
+    /**
+     * Information about the command
+     *
+     * @return The description of the command
+     */
+    @Override
+    public String getDescription() {
+        return "add network partition deployment";
+    }
+
+    /**
+     * This should return the syntax required for the command.
+     * <p/>
+     * Used to display help.
+     *
+     * @return The syntax for this command
+     */
+    @Override
+    public String getArgumentSyntax() {
+        return null;
+    }
+
+    /**
+     * The options accepted by the command
+     *
+     * @return The Options for the commands
+     */
+    @Override
+    public Options getOptions() {
+        return options;
+    }
+
+    /**
+     * Executing the commands. Returns a code
+     *
+     * @param context The context assoicated with the Command Line Application
+     * @param args    The arguments for the command
+     * @return The status code
+     * @throws org.apache.stratos.cli.exception.CommandException if any errors occur when executing the command
+     */
+    @Override
+    public int execute(StratosCommandContext context, String[] args) throws CommandException {
+        if (logger.isDebugEnabled()) {
+            logger.debug("Executing {} command...", getName());
+            logger.info("Get name" + getName());
```

Comment:
Info log available inside the isDebugEnabled() condition.Better to change that info log.